controllers/reconcile.go

```diff
@@ -132,6 +132,146 @@ func populateContainer(d *apps.Deployment, frontend *crd.Frontend, frontendEnvir
 	}
 }
 
+// getAkamaiSecret gets the akamai secret from the cluster
+func getAkamaiSecret(ctx context.Context, client client.Client, frontend *crd.Frontend) (*v1.Secret, error) {
+	secret := &v1.Secret{}
+	err := client.Get(ctx, types.NamespacedName{Name: "akamai", Namespace: frontend.Namespace}, secret)
+	if err != nil {
+		return nil, err
+	}
+	return secret, nil
+}
+
+// constructAkamaiEdgercFileFromSecret constructs the akamai edgerc file from the secret
+func makeAkamaiEdgercFileFromSecret(secret *v1.Secret) string {
+	edgercFile := "[default]\n"
```

Comment:
At the risk of just asking for work for works sake, which I would never do, a part of me sees it would be nice to have a function that takes a key, value as args and spits out the line. Better yet a function that takes a `map[string]string` and just spits them all out, writing just the section and that then doesn't feel so bad, but I'm open to ignoring that. I know I would do it if it were my PR, but I honestly don't mind either way.